### ISSUE TITLE
Command line option to emit list of linters

### ISSUE THREAD

**Issue by wrouesnel** [alecthomas/gometalinter#444]
I have tooling in my repositories which manages the vendoring and update of the linters used by gometalinter in a self-contained way. One of the big problems which maintaining this is I have to regex the paths of the linters out of metalinter's help text in order to stay automatically up to date with changes to metalinter.

It would help a lot of if there was a CLI option to emit the list of linters in JSON format.

**Comment by dnephin** [comment]
Could you tell us more about the tooling? Why is it necessary to manage the vendoring and updating of linters?  If you'd like to use `gometalinter` with your own linters (or specific versions of a linter) you could create a `.gometalinter.json` [config file](https://github.com/alecthomas/gometalinter/blob/master/config.go#L13-L62) to run just the linters you've installed. There is a section for [InstallFrom](https://github.com/alecthomas/gometalinter/blob/master/linters.go#L14-L21). Your tooling could read that the JSON config to install and update vendor. That way all the configuration is in the same place.

**Comment by wrouesnel** [comment]
So an example is over here: https://github.com/wrouesnel/postgres_exporter/

Briefly, I have gometalinter and all the linters vendors under a `tools` folder in each of my repos which uses them, and a build target which compiles the vendored tools. To maintain this stack, there is a separate target called "updatetools" which go get's metalinter, then runs it and extracts the list of linters currently being packaged as an authoritative source to follow what's going on with the community.

This model lets me ship the build system with minimal setup when someone else clones the repository, depending only (now) on a local golang install to get started.

But, I have to do a bunch of regex'ing on the metalinter output if I want to automatically track what new linters may have been added to it since last it was updated - and it's preferable to do this since worst case I get a new linter which I discover I want to disable, but best case some new tool now is finding problems for me in my code automatically.

Technically this would be solvable programmatically, but default linters is an unexported variable so I can't just have the build system import metalinter as a library and parse that (and I suppose not everyone is using go to build - certainly I only just switched from makefiles to magefiles).
